tools/cartopy_feature_download.py

```diff
@@ -64,6 +64,8 @@
         ('cultural', 'admin_0_pacific_groupings', '110m'),
         ('cultural', 'admin_1_states_provinces', '110m'),
         ('cultural', 'admin_1_states_provinces_lines', '110m'),
+        ('cultural', 'admin_1_states_provinces_lakes', '110m'),
+        ('cultural', 'admin_1_states_provinces_lakes', '50m'),
```

Comment:
```suggestion
        ('cultural', 'admin_1_states_provinces_lakes', ALL_SCALES),
```
It looks like all scales are available, so perhaps download all scales? Or, put these on the same line as `('110m', '50m')`